CODING_STANDARD.md

```diff
@@ -341,13 +341,12 @@ To avoid waiting until you've made a PR to find formatting issues, you can
 install clang-format locally and run it against your code as you are working.
 
 Different versions of clang-format have slightly different behaviors. CBMC uses
-clang-format-11 as it is available the repositories for Ubuntu 20.04 and
-Homebrew.
-To install on a Unix-like system, try installing using the system package
-manager:
+clang-format-15, available in the standard Ubuntu 24.04 and Homebrew
+repositories. To install on a Unix-like system, try installing using the
+system package manager:
 ```
-apt-get install clang-format-11  # Run this on Ubuntu, Debian etc.
-brew install clang-format@11     # Run this on a Mac with Homebrew installed
+apt-get install clang-format-15  # Run this on Ubuntu, Debian etc.
+brew install clang-format        # Run this on a Mac with Homebrew installed
```

Comment:
I'm not an expert on `brew`, but this seems like it will break in future when the default is upgraded (where as all the other places and previous code are explicit about the version).

Reply:
It seems Brew only has versioned variants for clang-format 8 and clang-format 11, while the latest one will install clang-format 19.